package.json

```diff
@@ -48,7 +48,7 @@
     "deepmerge": "3.2.0",
     "dispensary": "0.37.0",
     "es6-promisify": "6.0.1",
-    "eslint": "5.16.0",
+    "eslint": "6.0.1",
```

Comment:
Could you use the caret notation `^6.0.1`, such that we don't need to install multiple versions of eslint 6 in parger projects?